cmd/kubeadm/app/master/apiclient.go

```diff
@@ -206,14 +207,10 @@ func SetMasterTaintTolerations(meta *api.ObjectMeta) {
 	meta.Annotations[api.TolerationsAnnotationKey] = string(tolerationsAnnotation)
 }
 
-func SetMasterNodeAffinity(meta *api.ObjectMeta) {
+func SetNodeAffinity(meta *api.ObjectMeta, expr ...api.NodeSelectorRequirement) {
```

Comment:
We don't have documentation on public functions here and we should probably start.  Can you start documenting some of this stuff as you touch it?